src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react'
+import { Link } from 'react-router-dom'
+import NavButton from './NavButton'
+import styled from 'styled-components/macro'
+import { NavData } from './NavData'
+
+function Navigation() {
+  const [navigation, setNavigation] = useState(false)
+
+  function showNavigation() {
+    setNavigation(!navigation)
+  }
+
+  return (
+    <>
+      <NavButton showNavigation={showNavigation} navigation={navigation} />
+      <Nav navigation={navigation}>
+        <NavButton showNavigation={showNavigation} navigation={navigation} />
+        {NavData.map((item, index) => {
+          return (
+            <NavItem key={index}>
```

Comment:
The `key` should not be the `index` as the index of an item in an array can change and this is not good for react. Maybe go for the name or add an `id` to your data.

Reply:
Thanks, added an id to the data and used it as key in the map function.